<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRuleForm</class>
 <widget class="QDialog" name="AddRuleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить / изменить правило</string>
  </property>
  <widget class="QLabel" name="ConditionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Если:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ConditionLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>18</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ActionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>52</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>То:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ActionLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="OkCancelButtonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="isGlobalRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Глобальное</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="isLocalRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Локальное</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="elseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Иначе:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="elseLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
